richtextfx/src/main/java/org/fxmisc/richtext/model/SimpleEditableStyledDocument.java

```diff
@@ -22,14 +22,14 @@
 /**
  * Provides an implementation of {@link EditableStyledDocument}
  */
-public final class SimpleEditableStyledDocument<PS, S> implements EditableStyledDocument<PS, S> {
+public final class SimpleEditableStyledDocument<PS, SEG, S> implements EditableStyledDocument<PS, SEG, S> {
```

Comment:
Since we're maintaining source compatibility with `StyledTextArea`, why not do the same for this one? Unfortunately, I'm not sure what such a name would be called.